cmd/notation/verify.go

```diff
@@ -246,11 +246,13 @@ func getVerifier(ctx context.Context) (notation.Verifier, error) {
 	}
 	fileCache, err := crl.NewFileCache(cacheRoot)
 	if err != nil {
-		return nil, err
-	}
-	crlFetcher.Cache = &clicrl.CacheWithLog{
-		Cache:             fileCache,
-		DiscardCacheError: crlFetcher.DiscardCacheError,
+		// discard NewFileCache error directly as cache is not critical
+		fmt.Printf("Warning: %v\n", err)
```

Comment:
We are using this pattern to print out warnings in Notation CLI: https://github.com/search?q=repo%3Anotaryproject%2Fnotation+os.stderr&type=code
```suggestion
		fmt.Fprintf(os.Stderr, "Warning: %v\n", err)
```

Reply:
Updated.